libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1455,8 +1456,10 @@ void AP_OSD_Screen::draw_horizon(uint8_t x, uint8_t y)
 {
     AP_AHRS &ahrs = AP::ahrs();
     WITH_SEMAPHORE(ahrs.get_semaphore());
-    float roll = ahrs.roll;
-    float pitch = -ahrs.pitch;
+    float roll = 0;
+    float pitch = 0;
+    AP_Vehicle::get_osd_roll_pitch_rad(roll,pitch);
```

Comment:
Opps! Compilation error here
```suggestion
    AP::vehicle()->get_osd_roll_pitch_rad(roll,pitch);
```

Reply:
that was fixed locally, but could not push due to git "brownout"...now pushed